tests/unit/plugins/module_utils/mock_info_api.py

```diff
@@ -989,16 +989,27 @@ def get_providers_response(response_type):
             return resp
 
     @staticmethod
-    def get_users_response(response_type):
-        resp = [{
-                "name": "testuser",
-                }, {
-                "name": "testuser1",
-                }]
-        if response_type == "error":
-            return "Get Users List for PowerScale cluster: **.***.**.*** and access zone: System failed with error: SDK Error message"
+    def get_auth_users(response_type):
+        if response_type == "api":
```

Comment:
can't we merge this if else block into one like this:
```
return {
                "users": [
                    {
                        "id": "id1",
                        "id_name": "id_name1",
                        "name": "name1"
                    }
                ]
            }
```
it can be access like this

- For api: get_auth_users
- For module: get_auth_users['user']